mdn/yari
# feat(glean): add is_baseline metric to pings

## Summary



https://mozilla-hub.atlassian.net/browse/MP-542
https://bugzilla.mozilla.org/show_bug.cgi?id=1848187 - draft PR until the data review passes on this

### Problem



While we could track page views with a baseline status by manually exporting a list of pages, and feeding that into our metrics tools, this doesn't scale very well.

### Solution



Add an `is_baseline` metric to page views and events:
- `null` - no baseline status on the page
- `true` - the page is baseline
- `false` - the page is not baseline

Also update the CSP to allow sending glean pings from localhost:5042

---

## How did you test this change?

set:
```
REACT_APP_GLEAN_ENABLED=true
REACT_APP_GLEAN_DEBUG=true
REACT_APP_DEV_MODE=true
```

run: `yarn && yarn dev`

to test the null case:
- visit: http://localhost:5042/en-US/docs/Web/CSS/CSS_grid_layout/Masonry_layout
- observe: https://debug-ping-preview.firebaseapp.com/pings/mdn-dev/c01abaf4-6532-4557-bc9f-18bb893cb61b

to test the true case:
- visit: http://localhost:5042/en-US/docs/Web/CSS/flex
- observe: https://debug-ping-preview.firebaseapp.com/pings/mdn-dev/de5b13b2-d42c-4599-b9b6-be0ec5ad6d32

to test the false case:
- visit: http://localhost:5042/en-US/docs/Web/CSS/CSS_grid_layout/Subgrid
- observe: https://debug-ping-preview.firebaseapp.com/pings/mdn-dev/5237226d-53c6-44c2-b3ca-4acfe1852c86
